sdk/textanalytics/cognitiveservices-textanalytics/README.md

```diff
@@ -31,20 +47,73 @@ npm install @azure/cognitiveservices-textanalytics
 
 ### 2. Create and authenticate a `TextAnalyticsClient`
 
-TextAnalytics uses both AAD and service keys for authentication.
+TextAnalytics uses both AAD and subscription keys for authentication.
+
+#### Using a Subscription Key
+
+Use the [Azure CLI][azure_cli] snippet below to get the subscription key from the Text Analytics resource.
+
+```PowerShell
+az cognitiveservices account keys list --resource-group <your-resource-group-name> --name <your-resource-name>
+```
+
+Alternatively, you can get the endpoint and subscription key from the resource information in the [Azure Portal][azure_portal].
 
-Creating a client with a service key:
+Once you have a subscription key, you can use it as follows:
 
 ```js
+const {
+  TextAnalyticsClient,
+  CognitiveServicesCredential
+} = require("@azure/cognitiveservices-textanalytics");
+
 const client = new TextAnalyticsClient(
   "<endpoint>",
-  new CognitiveServicesCredentials("<service key>")
+  new CognitiveServicesCredential("<subscription key>")
 );
 ```
 
+#### Using an Azure Active Directory Credential
+
+Client subscription key authentication is used in most of the examples, but you can also authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below,
+or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to [register a new AAD application][register_aad_app] and grant access to Text Analytics by assigning the `"Cognitive Services User"` role to your service principal.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET.
```

Comment:
I would put these environment variable names in backtick quotes since they are verbatim.